### PR TITLE
portals: provisioning editor and tester enhandments

### PR DESCRIPTION
**Pull request by mmadariaga**
Fixed multiple break lines issue in provisioning template editor
Scaped xml alike templates in provisioning tester
Disabled warnings in provisioning tester
Requires klear to be updated

### CODE REVIEW

File: portals/application/controllers/KlearCustomRunCodeController.php [hunk 1]
```diff
@@ -82,8 +82,12 @@ protected function _runCode($type, $inputMac, $error = false)
                 exec($command, $output, $resultCode);
                 unlink($route . DIRECTORY_SEPARATOR . $filename);
 
-                $message = implode("<br>", $output); //ojo que aquí hemos quitado la llamada a gettext
+                array_walk($output, function (&$value, $key) {
+                    //Ensure xml's are readable in browser
+                    $value = htmlentities($value);
+                });
 
+                $message = implode("<br>", $output); //ojo que aquí hemos quitado la llamada a gettext
```
Comment (Kaian) [inline portals/application/controllers/KlearCustomRunCodeController.php]: _Ojo_ with this inline comment. Remove it please :policeman:

File: portals/application/configs/klear/TerminalModelsList.yaml [hunk 2]
```diff
@@ -127,7 +127,7 @@ production:
       labelOnEdit: true
       action: run-generic-code
       class: ui-silk-control-play-blue
-    runSpecificCode_dialog:
+    runSpecificCode_dialog: &runSpecificCode_dialog
```
Comment (Kaian) [inline portals/application/configs/klear/TerminalModelsList.yaml]: We usually call YAML links ending with **Link** word
Also, where is this link used :thinking: ?

Comment (mmadariaga) [inline portals/application/configs/klear/TerminalModelsList.yaml]: > We usually call YAML links ending with Link word
Ok, I'll replace some other dialogs names as well
> where is this link used?
Nowhere, although every single dialog declaration is referable.

**Comment by mmadariaga** [comment]
@cruzccl issue&commit message changed. Let me know wthat do you think about the new ones

**Comment by cruzccl** [comment]
I like it, ship it!

**Pull request merged**